docs/library/utime.rst

```diff
@@ -28,8 +28,8 @@ by following means:
   in such case).
 
 If actual calendar time is not maintained with a system/MicroPython RTC,
-functions below which require reference to current absolute time may
-behave not as expected.
+functions below which require reference to current absolute time may not
```

Comment:
> No, I'm using it in the sense of "will possibly not".

Then people don't understand you and get confused. Because you probably then mean the same as in "value of N % 4 may not reach a value of 4 for any N".

> "may behave not" is just wrong word order in English.

Said who? (Another "wrong" word order.)